README.md

```diff
@@ -18,6 +18,46 @@ These are examples of valid exercise names:
 - `javascript-2-b` : second part of exercise B for JavaScript.
 - `ruby-1-b` : first part of exercise B for Ruby.
 
+### Test messages
+
+Solving exercises that are part of the experiment is different from the regular website's process in two ways:
+
+1. Exercises are solved exclusively using in-browser coding.
+1. The test suite used to test the code is not available to the user.
+
+This means that when the code does _not_ pass one of the tests, the user won't know with what input the code was called. To help with this, each exercise should define a `.meta/test_messages.json` file.
```

Comment:
Add something like above then change this to?

```suggestion
To achieve with this, each exercise should define a `.meta/test_messages.json` file.
```

and then merge that with the paragraph below?